modules/algebra/src/main/java/org/eclipse/imagen/media/algebra/AddCRIF.java

```diff
@@ -0,0 +1,43 @@
+package org.eclipse.imagen.media.algebra;
```

Comment:
Copyright header, see the builders module. Noted only here, add to all new files.